extension/sql/vectorize--0.22.0--0.22.1.sql

```diff
@@ -0,0 +1,36 @@
+DROP FUNCTION IF EXISTS vectorize._handle_table_update(text, text[]);
+CREATE OR REPLACE FUNCTION vectorize._handle_table_update(
```

Comment:
I need to figure out what happens when the function moves from being pointed to a shared objected to just a pl/pgsql function.